pkg/tuned/controller.go

```diff
@@ -367,21 +440,56 @@ func ProfilesExtract(profiles []tunedv1.TunedProfile, recommendedProfile string)
 			if !change {
 				un = "un"
 			}
-			klog.Infof("recommended TuneD profile %s content %schanged [%s]", recommendedProfile, un, *profile.Name)
+			klog.Infof("profilesExtract(): recommended TuneD profile %s content %schanged [%s]", recommendedProfile, un, *profile.Name)
 		}
 
-		f, err := os.Create(profileFile)
+		err := os.WriteFile(profileFile, []byte(*profile.Data), 0644)
 		if err != nil {
-			return change, extracted, recommendedProfileDeps, fmt.Errorf("failed to create TuneD profile file %q: %v", profileFile, err)
-		}
-		defer f.Close()
-		if _, err = f.WriteString(*profile.Data); err != nil {
-			return change, extracted, recommendedProfileDeps, fmt.Errorf("failed to write TuneD profile file %q: %v", profileFile, err)
+			return change, "", extracted, recommendedProfileDeps, fmt.Errorf("failed to write TuneD profile file %q: %v", profileFile, err)
 		}
 		extracted[*profile.Name] = true
+		klog.V(2).Infof("profilesExtract(): extracted profile %q to %q (%d bytes)", *profile.Name, profileFile, len(*profile.Data))
+	}
+
+	profilesFP := profilesFingerprint(profiles, recommendedProfile)
+	klog.Infof("profilesExtract(): extracted profiles fingerprint: %s", profilesFP)
```

Comment:
Can we change this to something like:
_fingerprint of extracted profiles_
or (less preferably)
_extracted profiles' fingerprint_
It might things less confusing to cluster admins.

Reply:
sure, changed